sdk/src/main/java/com/microsoft/hydralab/performance/PerformanceInspectionResult.java

```diff
@@ -13,9 +13,10 @@ public class PerformanceInspectionResult {
     // TODO: restrict the size of it.
     public Object parsedData;
 
-    public PerformanceInspectionResult(File rawResultFile) {
+    public PerformanceInspectionResult(File rawResultFile, PerformanceInspection inspection) {
         this.timestamp = System.currentTimeMillis();
         this.rawResultFile = rawResultFile;
+        this.inspection = inspection;
```

Comment:
Why do we need to add this member` this.inspection`?
It does not seem to be used anywhere.

Reply:
Because we need to get the description, appId from inspection. For more details, please refer to `AndroidBatteryInfoResultParser`